### PR TITLE
[MIG] Migration of account_export_csv.

### PR DESCRIPTION
**Pull request by adrienpeiffer**

### CODE REVIEW

File: account_export_csv/wizard/account_export_csv_view.xml [hunk 1]
```diff
@@ -1,29 +1,45 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
     <data>
```
Comment (espo-tony) [inline account_export_csv/wizard/account_export_csv_view.xml]: data tag is not needed

**Review by espo-tony — CHANGES_REQUESTED** [review]
Some minor remarks.

**Comment by adrienpeiffer** [comment]
Thanks @espo-tony

**Review by espo-tony — APPROVED** [review]
:+1:

**Comment by sbidoul** [comment]
I get this on runbot when exporting journal entries.

```
Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/http.py", line 638, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 675, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 331, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 119, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 324, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 933, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 504, in response_wrap
    response = f(*args, **kw)
  File "/home/odoo/odoo-10.0/addons/web/controllers/main.py", line 889, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/odoo/odoo-10.0/addons/web/controllers/main.py", line 877, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 681, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 672, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/odoo/build/OCA/account-financial-reporting/account_export_csv/wizard/account_export_csv.py", line 229, in action_manual_export_journal_entries
    writer.writerows(rows)
  File "/home/odoo/build/OCA/account-financial-reporting/account_export_csv/wizard/account_export_csv.py", line 51, in writerows
    for row in rows:
  File "/home/odoo/build/OCA/account-financial-reporting/account_export_csv/wizard/account_export_csv.py", line 331, in _get_rows_journal_entries
    'date_end': self.date_end})
  File "/.repo_requirements/odoo/odoo/sql_db.py", line 141, in wrapper
    return f(self, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/sql_db.py", line 218, in execute
    res = self._obj.execute(query, params)
ProgrammingError: relation "account_move_reconcile" does not exist
LINE 32:           LEFT JOIN account_move_reconcile on
```

**Comment by melroy89** [comment]
could be directly worked further for v11.

File: account_export_csv/README.rst [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- Trial Balance
+- Analytic Balance (with accounts)
+- Journal Entries
+
+You can filter by date range
+
+Installation
+============
+
+This module depends on date_range module in thta can be found in `server-tools <https://github.com/OCA/server-tools/tree/10.0/date_range#readme>`_.
```
Comment (melroy89) [inline account_export_csv/README.rst:22]: small typo here. 'thta'

File: account_export_csv/tests/test_account_export_csv.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
```
Comment (melroy89) [inline account_export_csv/tests/test_account_export_csv.py:1]: utf-8 line can be removed in python3. Just safe the file as utf8 encoding is enough.

**Review by melroy89 — CHANGES_REQUESTED** [review]
small rework.

File: account_export_csv/tests/test_account_export_csv.py (hunk 3 above)
Comment (astirpe) [reply account_export_csv/tests/test_account_export_csv.py:1]: This PR is for Odoo 10.0 that still needs python2, so it is correct to keep this line. Will be removed when migrating the module to 11.0.

Comment (melroy89) [reply account_export_csv/tests/test_account_export_csv.py:1]: My bad!

**Comment by bealdav** [comment]
see https://github.com/OCA/account-financial-reporting/pull/597/commits/c295e79f35273cc2251b6ce9c653df4147f63345

**Pull request closed without merging**